gson/pom.xml

```diff
@@ -54,7 +54,13 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>3.8.2</version>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>pl.pragmatists</groupId>
```

Comment:
Any way you can implement this without adding another 3rd party dependency? We’ve worked hard to limit our dependencies. Gson is often embedded in other software & adding dependencies is a huge burden downstream.